pom.xml

```diff
@@ -57,39 +57,98 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.apache.felix</groupId>
-                <artifactId>maven-bundle-plugin</artifactId>
-                <extensions>true</extensions>
+                <groupId>biz.aQute.bnd</groupId>
```

Comment:
the plugin is already configured correctly in sling-bundle-parent. Please use the bnd.bnd file for customization.

Reply:
+1, done